allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
most of comments is beside the point, so nothing would have changed if i propose some of them separatelly, but i need all of them to use in my everyday work